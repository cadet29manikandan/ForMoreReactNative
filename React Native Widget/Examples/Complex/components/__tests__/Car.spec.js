import 'react-native';
import React from 'react';
import Car from '../Car';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <Car id={1} manufacturer="manufacturer" model="model" price="1 042 Kč" />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
