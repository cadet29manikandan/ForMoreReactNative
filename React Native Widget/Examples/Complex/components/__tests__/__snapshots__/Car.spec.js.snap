// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomWidth": 1,
      "borderColor": "darkgray",
      "flex": 1,
      "minHeight": 55,
      "paddingHorizontal": 16,
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "minHeight": 55,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={1}
        style={
          Object {
            "color": "#2d2d2d",
            "flex": 3,
            "fontSize": 16,
          }
        }
      >
        manufacturer model
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#2d2d2d",
              "flex": 3,
              "fontSize": 16,
            },
            Object {
              "flex": 1,
              "textAlign": "right",
            },
          ]
        }
      >
        1 042 Kč
      </Text>
    </View>
  </View>
</View>
`;
